internal/kafka/internal/services/util_test.go

```diff
@@ -23,6 +25,46 @@ const (
 	mockIDWithInvalidChars = "vp&xG^nl9MStC@SI*#c$6V^TKq0"
 )
 
+func buildKafkaDBApiRequest() dbapi.KafkaRequest {
+	route := []byte("unitesthost.com/v1/test-route")
+	return dbapi.KafkaRequest{
+		Region:                           "us-east-1",
+		ClusterID:                        uuid.NewString(),
+		CloudProvider:                    "AWS",
+		MultiAZ:                          false,
+		Name:                             "test-cluster",
+		Status:                           "Creating",
+		SsoClientID:                      uuid.NewString(),
+		SsoClientSecret:                  uuid.NewString(),
+		CanaryServiceAccountClientID:     uuid.NewString(),
+		CanaryServiceAccountClientSecret: uuid.NewString(),
+		SubscriptionId:                   "test",
+		Owner:                            "unit=test-user",
+		OwnerAccountId:                   uuid.NewString(),
+		BootstrapServerHost:              "unittesthost.com",
+		OrganisationId:                   "unit-test-org",
+		FailedReason:                     "",
+		PlacementId:                      "unit-test",
+		DesiredKafkaVersion:              "3",
+		ActualKafkaVersion:               "3",
+		DesiredStrimziVersion:            "0.27",
+		ActualStrimziVersion:             "0.27",
+		DesiredKafkaIBPVersion:           "v1",
+		ActualKafkaIBPVersion:            "v1",
+		KafkaUpgrading:                   false,
+		StrimziUpgrading:                 false,
+		KafkaIBPUpgrading:                false,
+		KafkaStorageSize:                 "5gi",
+		InstanceType:                     "developer",
+		QuotaType:                        "developer",
```

Comment:
```suggestion
		QuotaType:                        "ams",
```
Quota type is either `ams` or `quota-management-list`

Reply:
Thanks! changed to ams

Reply:
Thanks